src/evo/chainhelper.cpp

```diff
@@ -34,3 +37,24 @@ bool CChainstateHelper::HasChainLock(int nHeight, const uint256& blockHash) cons
 }
 
 int32_t CChainstateHelper::GetBestChainLockHeight() const { return clhandler.GetBestChainLock().getHeight(); }
+
+/** Passthrough functions to CInstantSendManager */
+std::optional<std::pair</*islock_hash=*/uint256, /*txid=*/uint256>> CChainstateHelper::HasConflictingISLock(
```

Comment:
nit: consider making this a struct

Reply:
Unlikely to house any functions and is used in a very limited manner (invoked twice in validation), think it's fine.

Reply:
nit: maybe rename it to `CChainstateHelper::ConflictingISLockIfAny(...)` ?

Reply:
Rename done, resolved in latest push!